types.d.ts

```diff
@@ -1,24 +1,5 @@
-interface PageInfo {
-  pageName: string;
-  pageLink: string;
-  pageTitle: string;
-  displayInNav: boolean;
-}
-
-interface BannerInfo {
-  title: string;
-}
-
-interface HeroIcons {
-  altText: string;
-  link: string;
-  path: string;
-}
 // Yaml Config
-// TODO: Make this only match the config.yaml file
 declare module '*.yaml' {
-  // Config
-  export const pageInfo: PageInfo[];
-  export const bannerInfo: BannerInfo;
-  export const heroIcons: HeroIcons[];
+  const value: unknown;
```

Comment:
can you explain why this is unknown?

Reply:
Done, the basic story here is that when we import the yaml file directly we do not know what the object looks out. We have a few options here we could say its `any` but thats bad we lose type compatibility. Alternatively we could say its an `object` but this api isnt very good, which leaves us with `unknown`. Given we do not want you to import `config.yaml` from any other file but `config.ts` making it unknown makes it harder to work with elsewhere without validating the types as well which is what we want.